### DISCUSSION TITLE
Do I still need MyASUS after getting G-Helper?

### DISCUSSION THREAD

**Discussion by Zolio** [seerge/g-helper#229] [Q&A]
I'm not sure whether I should still have the MyASUS app still installed or not

**Comment by ke1ne** [comment]
MyASUS is not required, however, it keeps multimedia buttons "default" work, sets battery limit additionally and gets some updates.
It' not necessary, in general, but I'm keeping it.

**Comment by thduc87** [comment]
so i think myasus also has OSD for the function keys as well, so if we remove myasus we'd lose all the osd hence i have a request for @seerge if possible to make the OSD included in g-helper, that would be great so we can all get rid of myasus.

**Reply by seerge** [reply]
@thduc87 hello , OSD by default is handled by asus optimization service + asus OSD apps, they exist independently from MyASUS. You can check this topic https://github.com/seerge/g-helper/discussions/168 and see how to enable OSD if it stopped working by some reason :)

**Reply by thduc87** [reply]
great this is exatly what i've been looking for the whole time, thanks @seerge
